spec/models/manageiq/providers/openstack/cloud_manager/refresher_rhos_juno_spec.rb

```diff
@@ -49,16 +49,16 @@
 
         assert_common
       end
-      ::Settings.ems.ems_refresh.openstack.is_admin = false
-      ::Settings.ems.ems_refresh.openstack_network.is_admin = false
+      ::Settings.ems_refresh.openstack.is_admin = false
```

Comment:
as commented in other PR, this should use 

https://github.com/Ladas/manageiq-providers-amazon/blob/7b1a7226dea0566fd17d3b06271782e213bf4969/spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb#L47

Best to clean them all up in a refactoring PR